src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let callCounter = 4;
```

Comment:
it shouldn't be limited. Start from 0 and just increase it on every function call